arches.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Not intended to be run directly.
+echo "  Linux i386..."
+GOOS=linux GOARCH=386 ./build.sh
+echo "  Linux amd64..."
+GOOS=linux GOARCH=amd64 ./build.sh
+echo "  Linux arm..."
+GOOS=linux GOARCH=arm64 ./build.sh
+echo "  Linux arm64..."
+GOOS=linux GOARCH=arm64 ./build.sh
+echo "  macOS amd64..."
+GOOS=darwin GOARCH=amd64 ./build.sh
+echo "  macOS arm64..."
+GOOS=darwin GOARCH=arm64 ./build.sh
+echo "  Windows 386..."
+GOOS=windows GOARCH=386 ./build.sh
```

Comment:
Is anybody running Windows 386?

If not I'd prefer not supplying binaries for it.

Reply:
I have heard it tends to be lighter weight in virtual machines and most projects I work on continue to support it but I don't mind dropping the line.